README.md

```diff
@@ -70,6 +70,31 @@ In `acts_as_list`, "higher" means further up the list (a lower `position`), and
 - `list_item.lower_item`
 - `list_item.lower_items` will return all the items below `list_item` in the list (ordered by the position, ascending)
 
+## Adding ActsAsList To An Existing Model
+As it stands ActsAsList requires position values to be set on the model before the instance methods above will work. Adding something like the below to your migration will set the default position. Change the parameters to order if you want a different initial ordering.
+
+```ruby
+class AddPositionToTodoItem < ActiveRecord::Migration
+  def change
+    add_column :todo_items, :position, :integer
+
```

Comment:
I tested this and it doesn't seem to be required - I guess `update_columns` just writes raw css